### PR TITLE
[BDRK-2975] Add new bdrk usage

### PR DESCRIPTION
**Pull request by jarvis57**
[BDRK-2975] 

Add new bdrk usage

Tested run: https://bedrock.basis-ai.com/project/view/bdrk-churn-prediction/training/pipeline/view/new-pipeline/run/2/step/train

Screenshot 2020-12-02 at 4 27 47 PM

[BDRK-2975]: https://basis-ai.atlassian.net/browse/BDRK-2975

### CODE REVIEW

File: doc/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+tensorflow
+bdrk[analyzer,model-monitoring]==0.7.0b1
+seaborn
+lightgbm
```
Comment (jarvis57) [inline doc/requirements.txt:4]: Added in this file to keep track of dependencies on notebook dev.

**Review by jarvis57 — COMMENTED** [review]
Pls help take a look at the notebook code, whether we can have a better example as Im not so familiar with the code.

**Comment by chiewwy** [comment]
@jarvis57 Some suggestions

1. Convert these model params to constants at the top of the cell and use `bdrk.log_param` to record them.
```
clf = lgb.LGBMClassifier(num_leaves=10, learning_rate=0.05, n_estimators=100)
```

2. Convert this part to `bdrk.log_metric` calls to produce the same graphs on Bedrock UI.
```
fig, axs = plt.subplots(1, 2, figsize=(12, 4))
lgb.plot_metric(clf, "roc_auc", ax=axs[0])
lgb.plot_metric(clf, "binary_logloss", ax=axs[1])
plt.show()
```

3. Use [`mlflow.lightgbm.save_model`](https://www.mlflow.org/docs/latest/python_api/mlflow.lightgbm.html#mlflow.lightgbm.save_model) to save `clf` to disk, then `bdrk.log_model` to upload to Bedrock.

4. Also show alternative approach using pickle + `log_model`, for models that aren't supported by MLflow Models. In the future we can extend this to use [`mlflow.pyfunc`](https://www.mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#workflows) too.

**Comment by jarvis57** [comment]
@chiewwy good idea. I will try taking a look at the current training code to see what is logged

**Comment by jarvis57** [comment]
Added more proper code, was following the things in the `train.py`
Sample run: https://bedrock.basis-ai.com/project/view/bdrk-churn-prediction/training/pipeline/view/new-pipeline/run/20/step/train

Screenshot 2020-12-03 at 2 04 15 PM

**Pull request merged**